Makefile

```diff
@@ -58,8 +58,11 @@ lint-and-test: check-generate-init lint test-unit
 
 # ---- Integration Tests (algod required) ---- #
 
+sandbox-dev-up: SHELL:=/bin/bash
 sandbox-dev-up:
 	docker-compose up -d algod
+	SANDBOX_READY_REGEX="^Last committed block:*" ; \
+	while ! [[ `docker compose exec algod goal node status | head -n 1` =~ $$SANDBOX_READY_REGEX ]] ; do sleep 1 ; done
```

Comment:
what these lines do is mainly querying `goal node status` and check first line has form `Last commit block: ...`, and keep retrying until the logic condition is satisfied.

Can it be more elegent?